litellm/types/utils.py

```diff
@@ -481,7 +482,7 @@ class Message(OpenAIObject):
     function_call: Optional[FunctionCall]
     audio: Optional[ChatCompletionAudioResponse] = None
     reasoning_content: Optional[str] = None
-    thinking_blocks: Optional[List[ChatCompletionThinkingBlock]] = None
+    thinking_blocks: Optional[List[ChatCompletionThinkingBlock | ChatCompletionRedactedThinkingBlock]] = None
```

Comment:
use Union[ChatCompletionThinkingBlock, ChatCompletionRedactedThinkingBlock] 

We need it to work on python 3.9 too